pkg/components/minio/minio.go

```diff
@@ -38,7 +38,6 @@ func Instance(mattermost *mattermostv1alpha1.ClusterInstallation) *minioOperator
 			},
 		},
 		Spec: minioOperator.MinIOInstanceSpec{
-			Version:     "RELEASE.2018-11-22T02-51-56Z",
```

Comment:
I don't see us setting the MinIO version anywhere else now. Is that an option they removed entirely?

Reply:
the replace to the Image field, but also they have a default value.